client/src/components/SignUpAd/SignUpAd.css

```diff
@@ -18,7 +17,6 @@
   margin: 0 5%;
 }
 
-
 .SignUpAd button {
```

Comment:
Here's another case where you're overwriting styles in another component. Can you go through and fix any other instances of this?